glean-core/src/glean.udl

```diff
@@ -498,3 +498,40 @@ interface EventMetric {
 
     i32 test_get_num_recorded_errors(ErrorType error, optional string? ping_name = null);
 };
+
+dictionary Rate {
+    i32 numerator;
+    i32 denominator;
```

Comment:
How I long for the day we can use unsigned types

Reply:
*side-eyes Kotlin*

Reply:
[a-c's up to Kotlin 1.6.10](https://github.com/mozilla-mobile/android-components/pull/12000), unsigned types are stable [as of 1.5](https://kotlinlang.org/docs/whatsnew15.html#standard-library) ......

Reply:
unsigned types are. literals are signed though, and the compiler will complain if you pass it `1` where it expects something unsigned.

BUT ... [turns out literals can now have a `u` suffix](https://kotlinlang.org/docs/basic-types.html#literals).

Time to revisit our APIs, but maybe _after_ this.